app/src/main/java/org/sopt/pingle/presentation/ui/joingroup/JoinGroupSuccessActivity.kt

```diff
@@ -0,0 +1,52 @@
+package org.sopt.pingle.presentation.ui.joingroup
+
+import android.os.Bundle
+import android.text.Spannable
+import android.text.SpannableString
+import android.text.style.ForegroundColorSpan
+import android.text.style.TextAppearanceSpan
+import androidx.core.content.ContextCompat
+import org.sopt.pingle.R
+import org.sopt.pingle.databinding.ActivityJoinGroupSuccessBinding
+import org.sopt.pingle.util.base.BindingActivity
+
+class JoinGroupSuccessActivity :
+    BindingActivity<ActivityJoinGroupSuccessBinding>(R.layout.activity_join_group_success) {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setDescriptionTextStyle()
+    }
+
+    private fun setDescriptionTextStyle() {
+        val spannableText = SpannableString(binding.tvJoinGroupSuccessDescription.text)
+
+        spannableText.setSpan(
+            TextAppearanceSpan(
+                this@JoinGroupSuccessActivity,
+                R.style.TextAppearance_Pingle_Sub_Semi_16
+            ),
+            START,
+            END,
+            Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
+        )
+
+        spannableText.setSpan(
+            ForegroundColorSpan(
+                ContextCompat.getColor(
+                    this@JoinGroupSuccessActivity,
+                    R.color.g_01
+                )
+            ),
+            START,
+            END,
+            Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
+        )
+        binding.tvJoinGroupSuccessDescription.text = spannableText
+    }
+
+    companion object {
+        const val START = 0
+        const val END = 3
```

Comment:
```suggestion
        const val GROUP_NAME_START = 0
        const val GROUP_NAME_END = 3
```
이런 식으로 작성해주는 게 더 직관적일 것 같네요.

근데 갑자기 생기는 궁금증,, 단체명은 3글자가 아닐 수도 있는데 이 부분 확인 하셨을까요? 스트링 추출 할 때도 동적 스트링 추출이 필요해 보입니당!

Reply:
와 이 부분은 어케 해야할까요 서버에서 내려온 단체명 스트링의 글자수만큼 숫자를 지정해줘야 하려나욤

Reply:
제 나름대로 코드짜서 올려놨어요!!